heron/instance/src/java/com/twitter/heron/instance/HeronInstance.java

```diff
@@ -237,8 +237,7 @@ private void handleException(Thread thread, Throwable exception) {
       if (exception instanceof Error) {
         LOG.log(Level.SEVERE,
             "Error caught in thread: " + thread.getName()
-                + " with thread id: " + thread.getId() + ". Process halting...",
```

Comment:
Just a clarification, this change basically doesn't log actual message whenever any error happens. Not only the OOM error.